src/backend/BindGroup.cpp

```diff
@@ -55,6 +55,10 @@ BindGroup::TextureInfo& BindGroup::TextureInfo::operator=(TextureInfo&& rhs)
         rhs.retainTextures();
         releaseTextures();
         textures = rhs.textures;
+        
+        //release the textures before clearning the vertor
+        for (auto& texture : rhs.textures)
+            CC_SAFE_RELEASE(texture);
```

Comment:
The destruction will release the textures, why need to release it here?

Reply:
It will be empty after you clean the texture vector, thus skip to release the textures

Reply:
got it